nifi-commons/nifi-record/src/test/java/org/apache/nifi/serialization/record/TestDataTypeUtils.java

```diff
@@ -442,6 +442,14 @@ private void testChooseDataType(List<DataType> dataTypes, Object value, DataType
         assertEquals(expected, actual);
     }
 
+    @Test
```

Comment:
Can we also please add a test to `TestAvroTypeUtil`:
```java
    @Test
    public void testConvertAvroMap() {
       // GIVEN
        Map<?, ?> expected = new HashMap<String, Object>() {{
            put(
                "nullableMapField",
                new HashMap<String, Object>() {{
                    put("key1", "value1");
                    put("key2", "value2");
                }}
            );
        }};

        Schema nullableMapFieldAvroSchema = Schema.createUnion(
            Schema.create(Type.NULL),
            Schema.create(Type.INT),
            Schema.createMap(Schema.create(Type.STRING))
        );

        Schema avroRecordSchema = Schema.createRecord(
            "record", "doc", "namespace", false,
            Arrays.asList(
                new Field("nullableMapField", nullableMapFieldAvroSchema, "nullable map field", (Object)null)
            )
        );

        Map<?, ?> value = new HashMap<Utf8, Object>(){{
            put(new Utf8("key1"), "value1");
            put(new Utf8("key2"), "value2");
        }};

        Record avroRecord = new GenericRecordBuilder(avroRecordSchema)
            .set("nullableMapField", value)
            .build();

        RecordSchema nifiRecordSchema = new SimpleRecordSchema(
            Arrays.asList(
                new RecordField("nullableMapField", RecordFieldType.CHOICE.getChoiceDataType(
                    RecordFieldType.MAP.getMapDataType(RecordFieldType.STRING.getDataType())
                ))
            )
        );

        // WHEN
        Object actual = AvroTypeUtil.convertAvroRecordToMap(avroRecord, nifiRecordSchema);

        // THEN
        assertEquals(expected, actual);
    }
```

Reply:
Good idea, done!